# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../mesa/timeb.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./src/mesa/drivers/dri/savage/savageioctl.c.orig	2010-05-03 16:24:07.000000000 -0400
+++ ./src/mesa/drivers/dri/savage/savageioctl.c	2010-05-03 16:24:46.000000000 -0400
@@ -43,7 +43,6 @@
 
 #include "drm.h"
 #include <sys/ioctl.h>
-#include <sys/timeb.h>
 
 #define DEPTH_SCALE_16 ((1<<16)-1)
 #define DEPTH_SCALE_24 ((1<<24)-1)
